bitnami/rabbitmq/README.md

```diff
@@ -152,7 +152,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `configuration`                              | RabbitMQ Configuration file content: required cluster configuration                                                                                                     | `""`                                              |
 | `tcpListenOptions.backlog`                   | Maximum size of the unaccepted TCP connections queue                                                                                                                    | `128`                                             |
 | `tcpListenOptions.nodelay`                   | When set to true, deactivates Nagle's algorithm. Default is true. Highly recommended for most users.                                                                    | `true`                                            |
-| `tcpListenOptions.linger.on`                 | Enable Server socket lingering                                                                                                                                          | `true`                                            |
+| `tcpListenOptions.linger.lingeron`           | Enable Server socket lingering                                                                                                                                          | `true`                                            |
```

Comment:
Hey @Lance-Drane, thanks for the contribution!

Against my issue comment and after see the result, I think we may prefer `lingerOn` to maintain the consistency  in camelCase. Could you update it?

Reply:
OK, fix is updated. Do you need me to squash the commits together?